### ISSUE TITLE
Add an way to have infinite hearts

### ISSUE THREAD

**Issue by VeryRandomMan** [alesan99/mari0_ae#593]
Add an way to make the custom characters have infinite hearts by doing something like this:
`"hearts": 0`

**Comment by WilliamFr0g** [comment]
Is it too hard to type 1e10 or 99999999 or something?

**Comment by VeryRandomMan** [comment]
> Is it too hard to type 1e10 or 99999999 or something?

No,but thing is,they still could die like that,yeah,it may take a lot to make an custom character lose 99999999 hearts but it's possible and that's the reason im suggesting this

**Comment by WilliamFr0g** [comment]
It would take about 6 days of standing in front of a laser

**Comment by WilliamFr0g** [comment]
And really, so what if they do that? They spent 6 days dying once when they could've thrown Mario down a pit thousands of times by then, it's really pretty insignificant

**Comment by greatbritdan** [comment]
Yeah you can't prevent death, just set the health really high. I'm not patching this
